com.microsoft.mrtk.uxcomponents/Tests/Runtime/MRTK.UXComponents.RuntimeTests.asmdef

```diff
@@ -10,7 +10,8 @@
         "Microsoft.MixedReality.Toolkit.Input.Runtime.Tests",
         "UnityEditor.TestRunner",
         "UnityEngine.TestRunner",
-        "Unity.XR.Interaction.Toolkit"
+        "Unity.XR.Interaction.Toolkit",
+        "Unity.TextMeshPro"
```

Comment:
I'm not sure if this gets resolved anywhere at this point, but should we add this to `msftTestDependencies`?

https://github.com/microsoft/MixedRealityToolkit-Unity/blob/c7cd8f800e5b866154b1c224cc57fb671db4e304/com.microsoft.mrtk.uxcomponents/package.json#L24-L26

Reply:
Hmm, probably!

Reply:
Done